diff --git a/dist/core/build/static-build.js b/dist/core/build/static-build.js
index 9474ee2b798e46ce74b13cbad2ce796e9d7882d2..d3d80bace090df8ad743aee7b2d62db50c7e9a74 100644
--- a/dist/core/build/static-build.js
+++ b/dist/core/build/static-build.js
@@ -89,9 +89,9 @@ async function staticBuild(opts, internals, ssrOutputChunkNames) {
     settings.timer.end("Static generate");
   } else if (settings.buildOutput === "server") {
     settings.timer.start("Server generate");
     await generatePages(opts, internals);
+    await ssrMoveAssets(opts);
     await cleanStaticOutput(opts, internals);
-    await ssrMoveAssets(opts);
     settings.timer.end("Server generate");
   }
 }
